docs/getstarted.md

```diff
@@ -17,83 +17,86 @@ This example creates a single cluster in a single VPC, then configures two route
 
-**Steps**
+### Steps
 
-   **Set up Service-to-Service communications**
+**Set up service-to-service communications**
 
-1. Create the Kubernetes Gateway `my-hotel`:
+1. Use aws cli to create a VPC Lattice service network, with the name `my-hotel`:
    ```bash
-   kubectl apply -f examples/my-hotel-gateway.yaml
-   ```
-   ***Note***: By default, the gateway (lattice service network) is not associated with cluster's VPC.  To associate a gateway (lattice service network) to VPC, `my-hotel-gateway.yaml` includes the following annotation.
-   ```
-   apiVersion: gateway.networking.k8s.io/v1beta1
-   kind: Gateway
-   metadata:
-     name: my-hotel
-     annotations:
-       application-networking.k8s.aws/lattice-vpc-association: "true"
+   aws vpc-lattice create-service-network --name my-hotel 
+   {
+    "arn": "<my-hotel-sn-arn>",
+    "authType": "NONE",
+    "id": "<my-hotel-sn-id>",
+    "name": "my-hotel"
+   }
    ```
-1. Verify that `my-hotel` gateway is created (this could take about five minutes):
+   
+1. Create the Lattice ServiceNetworkVpcAssociation between current k8s cluster VPC and `my-hotel` service network:
+   ```bash
+     aws vpc-lattice create-service-network-vpc-association --service-network-identifier <my-hotel-sn-id> --vpc-identifier <k8s-cluster-vpc-id>
+      {
+       "arn": "<snva-arn>",
+       "createdBy": "<timestamp>",
+       "id": "<snva-id>",
+       "status": "CREATE_IN_PROGRESS"
+       }
+      ```
+   
+   Wait until above ServiceNetworkVpcAssociation status change to `ACTIVE`:
    ```bash
-   kubectl get gateway  
-   ```
-   ```
-   NAME       CLASS                ADDRESS   READY   AGE
-   my-hotel   amazon-vpc-lattice                     7d12h
+   aws vpc-lattice  get-service-network-vpc-association --service-network-vpc-association-identifier snva-0041ace3a8658371e
+   {
+       ....
+       "status": "ACTIVE",
+   }
    ```
-1. Once the gateway is created, find the VPC Lattice service network:
+1. Create the Kubernetes Gateway `my-hotel`:
    ```bash
-   kubectl get gateway my-hotel -o yaml
-   ```
+   kubectl apply -f examples/my-hotel-gateway.yaml
    ```
-   apiVersion: gateway.networking.k8s.io/v1beta1
-   kind: Gateway
-   ...
-   status:
-   conditions:
-   message: 'aws-gateway-arn: arn:aws:vpc-lattice:us-west-2:694065802095:servicenetwork/sn-0ab6bb70055929edd'
-   reason: Reconciled
-   status: "True"
-   type: Schedules
+   Verify that `my-hotel` gateway is created with `PROGRAMMED` equals to `True`:
+   ```bash
+   kubectl get gateway  
+   
+   NAME       CLASS                ADDRESS   PROGRAMMED   AGE
+   my-hotel   amazon-vpc-lattice               True      7d12h
    ```
-1. Create the Kubernetes HTTPRoute rates for the parking service, review service, and HTTPRoute rate:
+
+1. Create the Kubernetes HTTPRoute `rates` that can has path match routes to the `parking` service and `review` service (this could take about a few minutes)
```

Comment:
Create the Kubernetes HTTPRoute `rates` that has a PathMatch routing to the `parking` service and `review` service (this could take about a few minutes)